### ISSUE TITLE
Accessibility - ChoiceGroup aria-labelledby field incorrect resulting in voiceover tonot announce text

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#3172]
### Bug Report

- __Package version(s)__: 4.31.0
- __Browser and OS versions__: macOS, Safari, voiceOver screen reader

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Planner

#### Describe the issue:

On macOS the screen reader does not read the UI text on ChoiceGroup controls. This is because the aria-labelledby field points to the wrong element. Narrator on Windows manages to find the text anyway.

#### Actual behavior:

The screen reader in macOS does not announce the UI text

#### Expected behavior:

The screen reader in macOS should announce the UI text

### Screenshot of incorrect field value

![image](https://user-images.githubusercontent.com/29383146/31830248-7e42b346-b574-11e7-97fa-4afafebfddb5.png)

**Comment by oengusmacinog** [comment]
Duplicate of Issue [#4209](https://github.com/OfficeDev/office-ui-fabric-react/issues/4209) and fixed by Pull Request [#4274](https://github.com/OfficeDev/office-ui-fabric-react/pull/4274)
